docs/example.rst

```diff
@@ -19,39 +16,33 @@ Example
       -[channel] *120*456#
 
 
-   Favourite Colour
-   ~~~~~~~~~~~~~~~~
+   ### Favourite Colour
    name: fav-colour
    type: choice
    save: colour
-
    question[multiple-choice]:`
      Hi {@msisdn}. What is your favourite colour?
-     1. Red {$red}
-     2. Blue {$blue}
+     1. Red {=red}
+     2. Blue {=blue}
```

Comment:
User testing suggested people expect the names in `routes` to also have `$`s:

```
routes:
   $red: chose-red
   $blue: chose-blue
```

I think an `=` will maybe make it clearer that `{=red}` is for mapping a choice to the symbol `red`, rather than to the symbol `=red`.